cypress/e2e/inventory/search/browse-search-permissions.cy.js

```diff
@@ -0,0 +1,53 @@
+import TopMenu from '../../../support/fragments/topMenu';
+import TestTypes from '../../../support/dictionary/testTypes';
+import InventorySearchAndFilter from '../../../support/fragments/inventory/inventorySearchAndFilter';
+import devTeams from '../../../support/dictionary/devTeams';
+import Users from '../../../support/fragments/users/users';
+import permissions from '../../../support/dictionary/permissions';
+
+describe('permissions: inventory', () => {
+    let firstUser;
+    let secondUser;
```

Comment:
I suggest to rename user to userWithAllPermissions and userWithOnlyViewPermissions

Reply:
Done